dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -238,3 +238,19 @@ discoveryPollingRate.desc=Rate in milliseconds at which the OpenID Connect clien
 
 forwardLoginParameter=Forward login parameter
 forwardLoginParameter.desc=Specifies a comma-separated list of parameter names to forward to the OpenID Connect provider. If a protected resource request includes one or more of the specified parameters, the OpenID Connect client will include those parameters and their values in the authorization endpoint request to the OpenID Connect provider.
+
+userApiSpec=User API specification
+userApiSpec.desc=Indicates the specification to use for the user API.
+
+# Do not translate "OpenShift", "TokenReview", "system:auth-delegator"
+userApiToken=Access token for user API calls
+userApiToken.desc=Specifies the access token that has permission to call the user API. For OpenShift, the token is from a service account with permission to call the TokenReview API, which usually requires the system:auth-delegator role.
+
+accessTokenAuthentication=Use access token authentication
+accessTokenAuthentication.desc=Determines whether the access token that is provided in the request is used for authentication. If true, the client must provide a valid access token.
+
+accessTokenAuthenticationSupported=Access token authentication supported
+accessTokenAuthenticationSupported.desc=Determines whether to support access token authentication if an access token is provided in the request. If true, and an access token is provided in the request, the access token will be used as an authentication token.
```

Comment:
@ayoho Thanks for making the updates. I see you pushed some more changes, and I just have one small suggested update. Let me know if you have any questions. 

Change
"If true, and an access token is provided in the request, the access token will be used as an authentication token."
To
"If true, and an access token is provided in the request, the access token **is** used as an authentication token."